<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>articles_management</class>
 <widget class="QDialog" name="articles_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Your Articles</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: qlineargradient(spread:pad, x1:0.497537, y1:1, x2:0.468443, y2:0.301, stop:0 rgba(209, 243, 252, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="header">
     <property name="styleSheet">
      <string notr="true">QFrame{
border-style: outset;
border-radius: 20px;
border-color: rgb(0, 240, 255);
border-width: 0.5px;
padding: 1px;
background-color: rgba(102, 255, 255, 50)
}
QLabel{
border: none;
background-color: rgba(0, 0, 0, 0)
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignLeft">
       <widget class="QFrame" name="banner">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="articles_management_logo">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>assets/icons/articles_management_logo.png</normaloff>assets/icons/articles_management_logo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="articles_management_title">
           <property name="styleSheet">
            <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
           </property>
           <property name="text">
            <string>Your Articles</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="buttons">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="add_articles_button">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>assets/icons/add_article.png</normaloff>assets/icons/add_article.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_articles_button">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>assets/icons/delete_article.png</normaloff>assets/icons/delete_article.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="articles_display">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="articles_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>595</width>
        <height>493</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
        border-radius: 1px;
        border-style: solid;
        border-color: rgb(179, 179, 179);
        border-width: 0.5px 0.5px 0.5px 0.5px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
